frontend/Movie_Pigeon/app/src/main/java/org/example/team_pigeon/movie_pigeon/eventCenter/AddMovieToMovieListEvent.java

```diff
@@ -0,0 +1,17 @@
+package org.example.team_pigeon.movie_pigeon.eventCenter;
+
+import org.example.team_pigeon.movie_pigeon.models.Movie;
+
+/**
+ * Created by SHENGX on 2017/2/21.
+ */
+
+public class AddMovieToMovieListEvent {
```

Comment:
The series of add, delete and update movie classes may be integrated into one class or as an interface to be implemented in the usage class.